hazelcast/src/test/java/com/hazelcast/nio/tcp/MemberReadHandlerTest.java

```diff
@@ -95,15 +95,15 @@ public void whenMultiplePackets() throws Exception {
         packet4.writeTo(buffer);
 
         buffer.flip();
-        reader.read(buffer);
+        readHandler.onRead(buffer);
 
         assertEquals(4, dispatcher.packets.size());
         assertEquals(packet1, dispatcher.packets.get(0));
         assertEquals(packet2, dispatcher.packets.get(1));
         assertEquals(packet3, dispatcher.packets.get(2));
         assertEquals(packet4, dispatcher.packets.get(3));
```

Comment:
I know it's not your code, and it's not the "heart" of this PR, but I've noticed it during the review and it's so ugly, so maybe one could change it anyway:
`assertEquals(asList(packet1, packet2, packet3, packet4), dispacher.packets);` would be much more readable.